src/main/webapp/app/localci/build-queue/build-queue.component.html

```diff
@@ -146,6 +155,17 @@ <h3 id="build-queue-running-heading" jhiTranslate="artemisApp.buildAgents.runnin
                             }
                         </ng-template>
                     </ngx-datatable-column>
+                    <ngx-datatable-column prop="jobTimingInfo.buildDuration" [minWidth]="150" [width]="150">
```

Comment:
Is there a reason we use fixed width here instead of relative?

Reply:
I used the same style used in the component (all the columns have fixed width). I agree that this needs rework, which will be addressed in future PRs.